janus_core/calculations/geom_opt.py

```diff
@@ -69,6 +69,13 @@ class GeomOpt(FileNameMixin):  # pylint: disable=too-many-instance-attributes
         Logger if log file has been specified.
     tracker : Optional[OfflineEmissionsTracker]
         Tracker if logging is enabled.
+
+    Methods
+    -------
+    set_optimizer()
+        Set optimizer for geometry optimization.
+    run()
+        Run geometry optimization.
```

Comment:
Does this raise duplicate definition warnings when building docs?

Reply:
Duplicate of the individual method docstrings?

They're optional, but this is just a list of the [methods for the class](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring), rather than the individual docstrings for each method.

Reply:
Often sphinx complains if things are fully documented in docstrings and also listed in `Methods` or `Attributes` blocks.

Reply:
Ah ok, it doesn't seem to be a problem in this case. Our single_point module already has a few listed: https://stfc.github.io/janus-core/apidoc/janus_core.html#module-janus_core.calculations.single_point

Reply:
Nope, doesn't raise an issue. All good. Weird though given I've had to "fix" so many of these in other docs. Maybe "Methods" doesn't do it in the same way "Attributes" does or maybe it depends on the Sphinx theme.